src/types/cloud.ts

```diff
@@ -110,6 +110,6 @@ export type InvoiceLineItem = {
 }
 
 export type SubscriptionStats = {
-	remaining_seats: number;
-	is_paid_tier: string;
+	remainingSeats: number;
```

Comment:
We can't change the names of these without changing what's being passed down from the server, since it's mapped using snake case naming. I think these names will need to be changed on the webapp PR instead.

Reply:
I created a PR for the same on the web-app repository and if you want me to remove these naming convention changes then it will not pass the type-checking.

Reply:
Agree with @devinbinnie -- until we go to the effort of renaming all the "incoming" fields from the server, we're stuck with the snack case here. Can we supply instructions to override the linter in this case?

Reply:
Just to confirm @aggmoulik, you're saying the linter is causing the error or the type checker? If it's the type checker, there's probably another instance that needs to be changed somewhere as well.

If it's the linter, then I think we need to change our ruleset to allow for snake case unless we plan on changing this convention app-wide.

Reply:
Having the server code return snake case was intentional way back when, so I don't think there's any plan to change that any time soon.

I'd say we could update the ESLint rules for that now, but we're going to be getting rid of mattermost-redux as a separate repository later this week, so I'd say to leave the casing as-is and use an `// eslint-ignore` wherever the warnings are thrown for the time being.